quipucords/quipucords/wsgi.py

```diff
@@ -16,9 +16,12 @@
 # pylint: disable=invalid-name
 application = get_wsgi_application()
 
-from scanner.manager import SCAN_MANAGER  # noqa: E402 pylint: disable=C0413
-
 from . import environment  # noqa: E402 pylint: disable=C0413
 
 environment.startup()
-SCAN_MANAGER.start()
+
+from scanner import manager  # noqa: E402 pylint: disable=C0413
+
+if not manager.SCAN_MANAGER:
+    manager.reinitialize()
+manager.SCAN_MANAGER.start()
```

Comment:
mmm, I'm under the impression the changes added to `quipucords/api/apps.py` are not necessary when factoring in this snippet :thinking: